HTML

```diff
@@ -0,0 +1,9 @@
+<!DocType <!DOCTYPE html>
```

Comment:
- this looks like a typo:
- `<!DocType <!DOCTYPE html>` should be  `<!DOCTYPE html>`

Reply:
reference:
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Introduction_to_HTML5#Declaring_that_the_document_contains_HTML5_mark-up_with_the_HTML5_doctype